wavin_ahc9000.py

```diff
@@ -375,6 +375,24 @@ def party_temperature(self, temperature: float):
     def desired_temperature(self) -> float:
         return self.__read_temp(16)
 
+class _ChannelCategory:
+    def __init__(self, channel: int, modbus: _WavinAHC9000Modbus):
+        self.__modbus = modbus
+        self.channel = channel
```

Comment:
I would like some sort of order between` _var`,` __var`, and normal `variables` but see that I totally failed to do so in the other classes. You can move self.channel to the top if you don't mind.